package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "replaywebpage",
   "productName": "ReplayWeb.page",
-  "version": "1.8.9",
+  "version": "1.8.10",
   "description": "Serverless Web Archive Replay",
   "repository": "https://github.com/webrecorder/replayweb.page",
   "homepage": "https://replayweb.page/",
   "author": "Webrecorder Software",
   "license": "AGPL-3.0-or-later",
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
-    "@webrecorder/wabac": "^2.16.10",
+    "@webrecorder/wabac": "github:webrecorder/wabac.js#add-index-html-cache",
```

Comment:
Should this point to a published wabac release before publishing replayweb.page?

Reply:
Yes, this is just for testing, since two PRs are tied. Will release that first and then update this once we've verified its all good.